### PR TITLE
Add Chinese translation for view

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: core/src/main/resources/hudson/model/View/newJobButtonBar.jelly [hunk 1]
```diff
@@ -23,6 +23,6 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/model/View/newJobButtonBar.jelly]: unrelated, but looks fine

File: core/src/main/resources/jenkins/model/Jenkins/newView.jelly [hunk 2]
```diff
@@ -33,7 +33,9 @@ THE SOFTWARE.
     <l:main-panel>
       <j:invokeStatic var="views" className="hudson.model.View" method="allInstantiable" />
       <n:form nameTitle="${%View name}" action="createView" copyTitle="${%Copy Existing View}"
-              descriptors="${views}" checkUrl="checkViewName" xmlns:n="/lib/hudson/newFromList" />
+              descriptors="${views}" checkUrl="checkViewName" xmlns:n="/lib/hudson/newFromList">
+        <button type="submit" id="ok-button">${%OK}</button>
```
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/model/Jenkins/newView.jelly]: Adds a button. not sure why

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
This pull requests does changes in addition to localization. They should be justified in the pull request or decoupled to a separate pull request

File: core/src/main/resources/hudson/model/View/newJobButtonBar.jelly (hunk 1 above)
Comment (LinuxSuRen) [reply core/src/main/resources/hudson/model/View/newJobButtonBar.jelly]: Ok than. We can keep them, or use some tools to clean all of them.

File: core/src/main/resources/jenkins/model/Jenkins/newView.jelly (hunk 2 above)
Comment (LinuxSuRen) [reply core/src/main/resources/jenkins/model/Jenkins/newView.jelly]: I add the button because of it is not localization. I will re-check this.

**Comment by LinuxSuRen** [comment]
Already fixes.

**Review by oleg-nenashev — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**